assets/sass/components/_buttons.scss

```diff
@@ -164,6 +164,15 @@ $buttonColor: get-color('brand', 'primary');
     }
 }
 
+/**
+ * Buttons: Warning Variant
+ */
+.btn.btn--warning {
```

Comment:
Do we need to add a new button style here for a one time use?

Reply:
Do you mean it should just be a new standalone style? I guess it's conceivable we'll want to put a button inside an error box at some stage...? It looked awful in the default pink (on the red border box)!

Reply:
Ah fair enough! No makes sense to have a new variant then